### PR TITLE
Create .quickstart.yml

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
Small PR to add quickstart.yml to be properly parsed by the Quickstart automation.

As this is just an addition of the quickstart.yml, there is no need for a new release or version bump.

### CODE REVIEW

File: .quickstart/quickstart.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+schema_key: iterable_schema
+
+dbt_versions: ">=1.3.0 <2.0.0"
+
+table_variables:
+  iterable__using_campaign_label_history:
+    - campaign_label_history
+  iterable__using_campaign_suppression_list_history:
+    - campaign_suppression_list_history
+
```
Comment (fivetran-reneeli) [inline .quickstart/quickstart.yml:11]: Seems like this was left out--

    iterable__using_user_unsubscribed_message_type_history:
        - user_unsubscribed_message_type

**Review by fivetran-reneeli — CHANGES_REQUESTED** [review]
One variable was left out!

File: .quickstart/quickstart.yml (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply .quickstart/quickstart.yml:13]: Good catch, just updated

**Comment by fivetran-joemarkiewicz** [comment]
@fivetran-reneeli thanks for reviewing, this is ready for re-review!

File: .quickstart/quickstart.yml (hunk 1 above)
Comment (fivetran-reneeli) [reply .quickstart/quickstart.yml:13]: Oh actually I just re checked and it looks like we have logic that checks for both `user_unsubscribed_message_type_history` and `user_unsubscribed_message_type`, so we should include both

Comment (fivetran-joemarkiewicz) [reply .quickstart/quickstart.yml:13]: @fivetran-reneeli `user_unsubscribed_message_type` is actually deprecated so I intentionally removed it. This is something we can remove down the line, but for quickstart we should exclude it.

**Review by fivetran-reneeli — APPROVED** [review]
approved!

**Pull request merged**